lib/merge-properties.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/**
+ * Shallowly merges each argument's own properties into a new object
+ * Does not modify source objects
+ * @param {...object} - the objects to be merged
+ * @returns {object}  - the new object
+ */
+module.exports = function() {
+  var target = {};
+
+  for (var index = 0; index < arguments.length; index++) {
+    var source = arguments[index];
+    for (var key in source) {
```

Comment:
`Object.keys(source).forEach()` is preferable to `for (var key in source)`

https://github.com/airbnb/javascript/issues/851